### ISSUE TITLE
Quickstart example crashes in google colab

### ISSUE THREAD

**Issue by VincentCCL** [UKPLab/sentence-transformers#398]
I was playing around with SBERT on Google Colab. While things were working before lunch, when I came back and reran the stuff, I got an error.
It is just the code from the Quickstart example, so I don't know what is wrong.
The google colab session is here: [https://colab.research.google.com/drive/1tevIPE1xmLwuPBcwDBNAFBRAu1EcFWht?usp=sharing](https://colab.research.google.com/drive/1tevIPE1xmLwuPBcwDBNAFBRAu1EcFWht?usp=sharing)

The code is:
```
from sentence_transformers import SentenceTransformer
model = SentenceTransformer('distilbert-base-nli-stsb-mean-tokens')

#Our sentences we like to encode
sentences = ['This framework generates embeddings for each input sentence',
             'Sentences are passed as a list of string.', 
             'The quick brown fox jumps over the lazy dog.']

#Sentences are encoded by calling model.encode()
sentence_embeddings = model.encode(sentences)

#Print the embeddings
for sentence, embedding in zip(sentences, sentence_embeddings):
    print("Sentence:", sentence)
    print("Embedding:", embedding)
    print("")
```
The error I get is:
```
/usr/local/lib/python3.6/dist-packages/transformers/tokenization_utils_base.py:1770: FutureWarning: The `pad_to_max_length` argument is deprecated and will be removed in a future version, use `padding=True` or `padding='longest'` to pad to the longest sequence in the batch, or use `padding='max_length'` to pad to a max length. In this case, you can give a specific length with `max_length` (e.g. `max_length=45`) or leave max_length to None to pad to the maximal input size of the model (e.g. 512 for Bert).
  FutureWarning,

---------------------------------------------------------------------------

IndexError                                Traceback (most recent call last)

<ipython-input-4-9bc6e799c403> in <module>()
      8 
      9 #Sentences are encoded by calling model.encode()
---> 10 sentence_embeddings = model.encode(sentences)
     11 
     12 #Print the embeddings

7 frames

/usr/local/lib/python3.6/dist-packages/transformers/modeling_distilbert.py in forward(self, input_ids)
    106             The embedded tokens (plus position embeddings, no token_type embeddings)
    107         """
--> 108         seq_length = input_ids.size(1)
    109         position_ids = torch.arange(seq_length, dtype=torch.long, device=input_ids.device)  # (max_seq_length)
    110         position_ids = position_ids.unsqueeze(0).expand_as(input_ids)  # (bs, max_seq_length)

IndexError: Dimension out of range (expected to be in range of [-1, 0], but got 1)

```

**Comment by nreimers** [comment]
Transformers just released a new version 3.1.0, which has several breaking changes.

Try to downgrade to transformers==3.0.2

**Comment by VincentCCL** [comment]
Thank you -- that worked.

!pip uninstall transformers -y
!pip install transformers==3.0.2
